source/fundamentals/connection/connection-options.txt

```diff
@@ -409,6 +409,14 @@ relevant options.
        | **Default**: ``null``
        | **Connection URI Example**: ``tlsDisableCertificateRevocationCheck=false``
 
+   * - **TranslationOptions**
+     - | Specifies options, such as the {+mdb-server+} version, for translating LINQ
+       | queries to the Query API.
```

Comment:
S: add a little more detail about what the effect is of setting a specific setting. The `ExpressionTranslationOptions` type could be explained more so users know which fields they can set, etc. Maybe a small example would suffice.

Reply:
Server version is currently the [only option](https://github.com/mongodb/mongo-csharp-driver/blob/main/src/MongoDB.Driver/ExpressionTranslationOptions.cs), but I phrased it openly since they anticipate more options later.

My understanding of that setting is that is translates the C# query to the Query API syntax appropriate to the given version. Do you think more info about that is helpful?

Reply:
Sure--but because it doesn't affect the user, and because it would likely take some effort to figure out, I'm going to open a follow-up ticket for it.

Reply:
sg